### ISSUE TITLE
Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense.VisualBasicCompletionCommandHandlerTests_XmlDoc.InvokeWithOverridableKeywordCommitSeeLangword failed

### ISSUE THREAD

**Issue by tmat** [dotnet/roslyn#23598]
https://ci.dot.net/job/dotnet_roslyn/job/master/job/windows_release_unit32_prtest/10999/testReport/junit/Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense/VisualBasicCompletionCommandHandlerTests_XmlDoc/InvokeWithOverridableKeywordCommitSeeLangword/

```
MESSAGE:
System.NullReferenceException : Object reference not set to an instance of an object.
+++++++++++++++++++
STACK TRACE:
at Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense.TestState.VB$StateMachine_40_AssertSelectedCompletionItem.MoveNext()
 --- End of stack trace from previous location where exception was thrown ---
 at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
 at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
at Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense.VisualBasicCompletionCommandHandlerTests_XmlDoc.VB$StateMachine_35_InvokeWithKeywordCommitSeeLangword.MoveNext()
 --- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) 
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
--- End of stack trace from previous location where exception was thrown ---
 at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
--- End of stack trace from previous location where exception was thrown --- 
at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
 at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
```

**Comment by chborl** [comment]
Seeing it here also: https://ci.dot.net/job/dotnet_roslyn/view/Official%20Builds/job/master-vs-deps/job/windows_release_unit32/13/

**Comment by chborl** [comment]
With callstack, 
```
MESSAGE:
System.Threading.Tasks.TaskCanceledException : A task was canceled.
+++++++++++++++++++
STACK TRACE:
at Roslyn.Utilities.TaskExtensions.WaitAndGetResult_CanCallOnBackground[T](Task`1 task, CancellationToken cancellationToken) at 
Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.Completion.Controller.Session.WaitForModel_DoNotCallDirectly(Boolean blockForInitialItems) at 
Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.Completion.Controller.WaitForModel() at 
Microsoft.CodeAnalysis.Editor.Implementation.IntelliSense.Completion.Controller.Microsoft.CodeAnalysis.Editor.ICommandHandler<Microsoft.CodeAnalysis.Editor.Commands.CommitUniqueCompletionListItemCommandArgs>.ExecuteCommand(CommitUniqueCompletionListItemCommandArgs args, Action nextHandler) at 
Microsoft.CodeAnalysis.Editor.CommandHandlers.AbstractCompletionCommandHandler.ExecuteCommandWorker[TCommandArgs](TCommandArgs args, Action nextHandler) at 
Microsoft.CodeAnalysis.Editor.CommandHandlers.AbstractCompletionCommandHandler.Microsoft.CodeAnalysis.Editor.ICommandHandler<Microsoft.CodeAnalysis.Editor.Commands.CommitUniqueCompletionListItemCommandArgs>.ExecuteCommand(CommitUniqueCompletionListItemCommandArgs args, Action nextHandler) at 
Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense.TestState._Closure$__27-0._Lambda$__0(CommitUniqueCompletionListItemCommandArgs a, Action n) at 
Microsoft.CodeAnalysis.Editor.UnitTests.AbstractCommandHandlerTestState.SendCommitUniqueCompletionListItem(Action`2 commandHandler, Action nextHandler) at 
Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense.TestState.SendCommitUniqueCompletionListItem() at 
Microsoft.CodeAnalysis.Editor.UnitTests.IntelliSense.VisualBasicCompletionCommandHandlerTests_XmlDoc.VB$StateMachine_35_InvokeWithKeywordCommitSeeLangword.MoveNext() --- End of stack trace from previous location where exception was thrown --- at 
System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at 
System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
--- End of stack trace from previous location where exception was thrown --- at 
System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at 
System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task) 
--- End of stack trace from previous location where exception was thrown --- at 
System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task) at 
System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)```

**Comment by chborl** [comment]
@jinujoseph , per Jason we should have someone look at this quickly

**Comment by sharwell** [comment]
Fixed in #23641
